src/com/walmartlabs/lacinia/pedestal.clj

```diff
@@ -362,14 +362,19 @@
   :graphiql (default false)
   : If true, enables routes for the GraphiQL IDE.
 
+  :interceptors
+  : The map of interceptors to be passed to [[routes-from-interceptor-map]].
+    If not provided, [[graphql-interceptors]] is invoked.
+
```

Comment:
Previously, you were expected to generate routes from customized interceptors and that is still supported, but most applications will just want to customize the interceptors.